lib/rubocop/cop/lint/empty_conditional_body.rb

```diff
@@ -108,7 +108,7 @@ def correct_other_branches(corrector, node)
         end
 
         def require_other_branches_correction?(node)
-          return false unless node.if_type? && node.else_branch
+          return false unless node.if_type? && (node.else_branch || node.else?)
```

Comment:
`node.else_branch` looks unnecessary. Is it necessary?

```suggestion
          return false unless node.if_type? && node.else?
```